app/views/liquids/_dotnet_fiddle.html.erb

```diff
@@ -0,0 +1,13 @@
+<%# Weird hack due to the fact that embedded links have Widget in between.
+Starting from https:// we end at 24 characters at dotnetfiddle.net %>
+
+<iframe
+  src="<%= link.insert(24, "/Widget").to_s %>"
```

Comment:
can we keep this "hack" in the liquit tag ruby file? We should also test it :D 

Is there a chance the number of chars will change?

Reply:
➕ for moving this into the actual tag instead of keeping it in the view. I think the 24 is unlikely to change, as it's the lenght of "https://dotnetfiddle.net", but I think a more robust solution with Ruby's `URI` module for building the link would be preferable.

Reply:
Sure, thanks for the inputs everyone. I had the "hack" (xD) shifted in the controller, but I sticked to using the string.insert method. 

I'll try to make the change through the URI module now. Thanks for the heads up :+1